### PR TITLE
Upgrade massively to be compatible with Ghost 3.1

### PR DESCRIPTION
**Pull request by Gwendal-R**
Upgrade massively theme to be compatible with Ghost 3.1
Change error.code to error.statusCode
Add new style to handle Koenig Bookmark card.
Upgrade version number in package.json and define new version compatibility.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,11 @@
             "name": "ajlkn",
             "email": "aj@lkn.io",
             "url": "https://twitter.com/ajlkn"
-        }
+        },
+        {
+            "name": "Gwendal R",
+            "email": "contact@gwendal-raballand.fr",
+            "url": "https://github.com/Gwendal-R"
```
Comment (zfdev) [inline package.json]: Are you the author for this theme?

Comment (Gwendal-R) [reply package.json]: I’m not author for this theme but a contributor as I make domes changes in order to assure compatibility with Ghost v3.

If you think I don’t have to put my name on project I can edit file and remove it.

**Comment by jt28828** [comment]
Just wanted to say thanks for these additions, I copied your CSS / handlebar changes into my local massively theme and it completely fixed bookmark styling. 
In its current state, without this pull request the theme isn't usable with bookmark cards as they fill a huge chunk of the blog they're in and break the styling on the page underneath them.

**Comment by minimaluminium** [comment]
Thanks for your work, but there are couple of issues in this PR.

- New CSS was added directly to the generated file, not SASS files.
- Contributor list shouldn't be changed
- Missing bookmark styles were added in https://github.com/TryGhost/Massively/commit/b821e05886c971e1a9d74ba98c246ff4d2f17724

`ghost` and `ghost-api` versions were bumped in https://github.com/TryGhost/Massively/commit/862a4839382e74cfbf1a10c4f6400493e69faca9, so closing this PR.

**Pull request closed without merging**